lib/pbench/server/api/resources/models.py

```diff
@@ -16,7 +16,7 @@ class UserModel(Base):
     lastName = Column(String(255), unique=False, nullable=False)
     password = Column(String(255), nullable=False)
     registered_on = Column(DateTime, nullable=False)
-    email = Column(String(255), unique=False, nullable=False)
+    email = Column(String(255), unique=True, nullable=False)
```

Comment:
Interesting question: unless we're using email as our access key, it doesn't really need to be unique... although having the same contact for multiple users isn't exactly useful, either.

Reply:
I thought an email should be unique and have 1-1 mapping with username, so a user can login with either username or email